templates/company/company_name_availability/form.html.tx

```diff
@@ -1,4 +1,4 @@
-% cascade base { title=> "Company Name Availability Checker - Companies House", classes=>"profile", disable_header_search => 1 }
+% cascade base { title=> $query ~" Company name availability checker - Find and update company information - GOV.UK", classes=>"profile", disable_header_search => 1 }
```

Comment:
With the others in this PR now being lower case, should this be `company name availability checker`?

Reply:
Ignore this comment, just spotted it follows a different pattern to the ones with lower cases